src/migrations/components/blockThreeColumnsFeatures.ts

```diff
@@ -144,6 +150,28 @@ export const getBlockThreeColumnsFeaturesMigration: ContentfulMigrationGenerator
 
                 blockThreeColumnsFeatures.editField("entries").validations([{ size: { max: 12 } }]);
             },
+            4: migration => {
+                const blockThreeColumnsFeaturesEntry = migration.editContentType(
+                    "blockThreeColumnsFeaturesEntry"
+                );
+
+                blockThreeColumnsFeaturesEntry.createField("backIcon", {
+                    type: "Link",
+                    name: t.blockThreeColumnsFeaturesEntry.fields.backIcon,
+                    linkType: "Asset",
+                    validations: [{ linkMimetypeGroup: ["image"] }],
+                });
+
+                blockThreeColumnsFeaturesEntry.createField("backHeadline", {
+                    type: "RichText",
+                    name: t.blockThreeColumnsFeaturesEntry.fields.backHeadline,
+                });
+
+                blockThreeColumnsFeaturesEntry.createField("backText", {
+                    type: "RichText",
+                    name: t.blockThreeColumnsFeaturesEntry.fields.backText,
+                });
```

Comment:
The validation for the RTE is missing. Validations on RTE fields set which type of RTE content is allowed (e.g. Headlines, bold, italic, ...)